vars/runSctTest.groovy

```diff
@@ -27,6 +27,14 @@ def call(Map params, String region, functional_test = false, Map pipelineParams
     export SCT_CONFIG_FILES=${test_config}
     export SCT_COLLECT_LOGS=false
 
+    if [[ -n "${params.stress_duration ? params.stress_duration : ''}" ]] ; then
+        export SCT_STRESS_DURATION=${params.stress_duration}
+    fi
+
+    if [[ -n "${params.prepare_duration ? params.prepare_duration : ''}" ]] ; then
```

Comment:
```suggestion
    if [[ -n "${params.prepare_duration ? params.prepare_duration : '0'}" ]] ; then
```

Reply:
Same as above, if it is not set we don't need to export env variable, checks for values other than interger, will failed job earlier on time calculation. Additional check will be add to sct_config validator

Reply:
this is fine and follow the logic how we use env variables across the pipeline